docs/src/app_dev/app_val_tokenomics/index.rst

```diff
@@ -99,28 +112,10 @@ More detail is provided in the following sections.
 The following use cases are used in explaining the application and
 validator tokenomics below:
 
--  :ref:`cc_xfer_nomics`
-
--  :ref:`feat_app_act_marker_nomics`
-
--  :ref:`cc_xfer_splice_wallet_nomics`
-
--  :ref:`traffic_nomics`
-
--  :ref:`val_live_nomics`
-
--  :ref:`sv_live_nomics`
-
-The last section is a :ref:`faq_nomics`.
-
-
 .. toctree::
-   :hidden:
-
-   cc_xfer_nomics
-   feat_app_act_marker_nomics
-   cc_xfer_splice_wallet_nomics
-   traffic_nomics
-   val_live_nomics
-   sv_live_nomics
-   faq_nomics
+   cc_transfer_tokenomics
```

Comment:
Looking at the top-level ToC it seems to me that the tokenomics section would fit better under "Background". This would also solve the confusion stemming from having both "Overview" and "Tokenomics Overview". It also solves the problem that tokenomics also talks about validator tokenomics.

Screenshot for reference:
<img width="1920" height="2125" alt="image" src="https://github.com/user-attachments/assets/dc313be8-e712-472f-a5bb-b4c8f90f62ee" />

Reply:
Done